doc/source/whatsnew/v1.5.0.rst

```diff
@@ -499,6 +507,14 @@ Other API changes
 Deprecations
 ~~~~~~~~~~~~
 
+.. warning::
+
+    In the next major release, 2.0, several larger API changes are being considered without a formal deprecation such as
```

Comment:
Not sure at that moment we could commit to the next release being 2.0. (we need to discuss the status of the deprecations)

Reply:
This does say "next major release" but I think would still need some clarification.

Reply:
i tried to move the discussion about deprecations to RLS 2.0 issue https://github.com/pandas-dev/pandas/issues/46776#issuecomment-1099087762 from the RLS 1.5 issue #45223 just in case the next release is 1.6

cc @datapythonista

Reply:
I clarified to "next major version" release just to reference that it will happen in 2.0 but not infer that 2.0 will be the next release.